src/kubeobjects/virtualservice.go

```diff
@@ -1,16 +1,16 @@
-package istio
+package kubeobjects
```

Comment:
this was good where it was,
a `VirtualService` is an isito specific thingy, should stay "locked away" in the `istio` package 😛